charts/ziti-controller/templates/deployment.yaml

```diff
@@ -132,10 +132,19 @@ spec:
                 secretKeyRef:
                   name:  {{ include "ziti-controller.fullname" . }}-admin-secret
                   key: admin-password
+            # Add additional environment variables
             {{- range $key, $val := .Values.env }}
             - name: {{ $key | quote }}
               value: {{ $val | quote }}
             {{- end }}
+            # Add additional secrets as environment variables
+            {{- range .Values.secrets }}
```

Comment:
Let's name this to stand out as it relates to the existing top-level `env` dictionary. How about `envSecrets`?